examples/scenarioSatelliteConstellation.py

```diff
@@ -0,0 +1,292 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2016, Autonomous Vehicle Systems Lab, University of Colorado at Boulder
```

Comment:
The year needs to change in this copyright statement

Reply:
done